_layouts/wmt_event.html

```diff
@@ -0,0 +1,223 @@
+---
+layout: default
+parent: Events
+---
+
+<h1>{{ page.title }}</h1>
+<h2>{{ page.description }}</h2>
+<hr />
+
+{% assign current_date = site.time | date: "%Y-%m-%d" %}
+
+<p>{% if current_date < page.end_date %}{{ page.future_tense_opening_paragraph }}{% else %}{{ page.past_tense_opening_paragraph }}{% endif %}</p>
```

Comment:
Really we should do this in JavaScript so that we don't need to rebuild the site for it to be correct.